code/src/index.css

```diff
@@ -11,3 +15,130 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.MovieContainer {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+}
+
+.MovieWrapper {
+  position: relative;
+  width: 100%;
+  background-color: #000;
+}
+
+.MovieImage {
+  height: 600px;
+  width: 100%;
+  object-fit: cover;
+}
+
+.MovieOverlay {
+  color: #fff;
+  padding: 15px;
+  word-wrap: break-word;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background: linear-gradient(rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 1));
+}
+
+.MovieTitle {
+  margin: 0;
+  font-size: 25px;
+}
+
+.releasedate {
+  text-transform: uppercase;
+}
+
+.home {
+  position: fixed;
+  margin: 20px;
+  padding: 15px;
+  font-size: 25px;
+  margin-left: 50px;
+  background: none;
+  border: 3px solid white;
+  transition: transform 0.3s;
+  cursor: pointer;
+  text-transform: uppercase;
+}
+
+.vote {
+  color: red;
+}
+
+.overview {
+  font-weight: 400;
+}
+
+.detail {
+  margin-left: 20px;
+}
+
+.home:hover {
+  transform: scale(1.3);
+  background: black;
+  color: white;
+  border: none;
+}
+
+h1 {
+  color: white;
+  max-width: 540px;
+}
+
+.details {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  padding: 50px;
+}
+
+.detailsimage {
+  border: solid 5px white;
+  width: 20%;
+  min-width: 400px;
```

Comment:
Try to remove the min-width to avoid to end up outside the window in the mobile version. Also change the width to maybe a higher amount in %.